modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/Inbox.java

```diff
@@ -365,7 +365,9 @@ private void onNodeLeft0(String nodeName) throws Exception {
         checkState();
 
         if (perNodeBuffers.get(nodeName).check() != State.END) {
-            throw new IgniteInternalCheckedException(NODE_LEFT_ERR, "Failed to execute query, node left [nodeName=" + nodeName + ']');
+            throw new IgniteInternalCheckedException(
```

Comment:
Let's introduce NodeLeftException and pass it as cause.
Thus, in trace user will see `RUNTIME_EXECUTION_ERR` code and `NODE_LEFT_ERR` code as a root cause.

Reply:
Yeah, such error is raised three times.